src/HW_2_OOP/SlytherinStudents.java

```diff
@@ -0,0 +1,43 @@
+package HW_2_OOP;
+
+public class SlytherinStudents extends HogwartsStudentGeneral {
```

Comment:
```suggestion
public class SlytherinStudent extends HogwartsStudentGeneral {
```
классы нужно называть в единственном числе